Misc/NEWS.d/next/Core and Builtins/2021-03-19-08-16-51.bpo-43551._a2GcM.rst

```diff
@@ -0,0 +1 @@
+fix PyImport_Import use static silly_list, under building Python with --with-experimental-isolated-subinterpreters share silly_list in multi sub interpreters cause crash.
```

Comment:
--with-experimental-isolated-subinterpreters option is supposed to remain secret!

Either remove your NEWS entry, or just say "subinterpreters". For example, "Fix the import function for subinterpreters." I'm not sure that it's worth it to document every single subinterpreter fix.